graphics/nxwidgets/src/cprogressbar.cxx

```diff
@@ -181,7 +181,7 @@ void CProgressBar::drawContents(CGraphicsPort *port)
 
   if (m_showPercentageText)
     {
-      char text[6];
+      char text[8];
```

Comment:
Why this is 8 and not 12? I mean that 11 for maximum chars for `int` + 1 for `%`. Just for me to understand. Maybe we can use `%hd%%` or `%hhd%%`?

Reply:
Done.